Documentation/intro-to-cli.md

```diff
@@ -47,7 +47,41 @@ You can get a sense of using the tools from the examples below.
 Design
 ======
 
-More content here.
+There are couple of moving pieces that you make up the general design of the .NET Core CLI:
+
+* The `dotnet` driver
+* Specific commands that are part of the package
+
+The `dotnet` driver is very simple and its primary role is to run the commands and give users basic information about usage. 
```

Comment:
I think it may be better to just say "commands" instead of "the commands".  My first reaction when I read this sentence was, "what commands?".